lib/dogapi/facade.rb

```diff
@@ -296,6 +297,29 @@ def graph_snapshot(metric_query, start_ts, end_ts, event_query = nil)
     end
 
     #
+    # EMBEDS
+    #
+    def get_all_embeds()
+      @embed_svc.get_all_embeds()
+    end
+
+    def get_embed(embed_id, size="medium", legend="no", template_vars= {})
+      @embed_svc.get_embed(embed_id, size, legend, template_vars)
+    end
+
+    def create_embed(graph_json, timeframe="1_hour", size="medium", legend="no", title="Embed created through API")
```

Comment:
Is the `title` parameter mandatory ? 

From [documentation](http://docs.datadoghq.com/api/#embeds) page, it looks like it's automatically set when no value is submitted, i.e.:
```
curl -POST \
    -d 'graph_json={"requests":[{"q":"avg:system.load.1{*}"}],"viz":"timeseries","events":[]}' \
    -d "timeframe=1_hour" \
    -d "size=medium" \
    -d "legend=yes" \
    "https://app.datadoghq.com/api/v1/graph/embed?api_key=${api_key}&application_key=${app_key}"
```
leads to 
```
{   
  "embed_id": "5f585b01c81b12ecdf5f40df0382738d0919170639985d3df5e2fc4232865b0c", 
  "template_variables": [], 
  "html": "<iframe src=\"https://app.datadoghq.com/graph/embed?token=5f585b01c81b12ecdf5f40df0382738d0919170639985d3df5e2fc4232865b0c&height=300&width=600&legend=true\" width=\"600\" height=\"300\" frameBorder=\"0\"></iframe>", 
  "graph_title": "Embed created through API", 
  "revoked": false, 
  "dash_url": null, 
  "shared_by": 3658, 
  "dash_name": null
```
(note the `graph_title` field value)